lib/screens/shorts/shorts_main.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:brunstadtv_app/components/loading_generic.dart';
+import 'package:brunstadtv_app/components/shorts/shorts_videos_stream_player.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:brunstadtv_app/providers/shorts_videos_service.dart';
+
+class ShortsMain extends ConsumerStatefulWidget {
+  const ShortsMain({Key? key}) : super(key: key);
+
+  @override
+  ConsumerState<ShortsMain> createState() => _ShortsMainState();
+}
+
+class _ShortsMainState extends ConsumerState<ShortsMain> {
+  @override
+  void initState() {
+    SystemChrome.setPreferredOrientations([
+      DeviceOrientation.landscapeRight,
+    ]);
+    ref.read(streamsForEpisodeProvider);
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final List<ShortsVideo>? shortsVideos = ref.watch(streamsForEpisodeProvider).valueOrNull;
```

Comment:
If you remove the `valueOrNull` you get an `AsyncValue`, which has a nice `when` function to build your widget based on the future state (loading, error, data, etc).
See https://riverpod.dev/docs/providers/future_provider